mkosi/__init__.py

```diff
@@ -2525,10 +2525,10 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
         return bump_image_version()
 
     if args.verb == Verb.summary:
-        text = ""
-
-        for config in presets:
-            text += f"{summary(args, config)}\n"
+        if args.json:
+            text = "\n".join(p.to_json() for p in presets)
```

Comment:
I think I'd rather see something like this (though quick and hacky)
```suggestion
            payload = ",".join(p.to_json() for p in presets)
            text = f'{"Presets": [{payload}]}'
```
instead of newline separated JSON objects. This way we don't box ourselves into a corner right away and keep what we output here extensible.